src/features/product/services/get-all-products.ts

```diff
@@ -11,7 +11,7 @@ type ServiceType = (params?: Params) => Promise<Response>
 const getAllProductsService: ServiceType = async (params) => {
   const filters = [
     params?.category && `categoryId=${params.category}`,
-    params?.search && `name_like=${params.search}`,
+    //params?.search && `name_like=${params.search}`,
```

Comment:
Entendi como resolveu aqui, mas tirou a lógica que estava pro searchInput, talvez seja melhor usar uma hook própria da página de detalhes do produto. Eu e o Dan adicionamos uma no merge anterior, se quiser trocar uma idéia pra refatorar esse PR chama nóis!